lib/charms/loki_k8s/v0/loki.py

```diff
@@ -90,6 +270,84 @@ def unit_ip(self) -> str:
             return str(bind_address)
         return ""
 
+    def _remove_alert_rules_files(self, container) -> None:
+        """Remove alert rules files"""
+
+        for f in container.list_files(RULES_DIR):
+            container.remove_path(f.path, recursive=True)
+            logger.debug("Alert rule file '%s' deleted", f.path)
```

Comment:
Could also be a folder?

```suggestion
            logger.debug("Deleting alert rule item '%s'", f.path)
            container.remove_path(f.path, recursive=True)
```

Reply:
No should be folders there. Fixed